### ISSUE TITLE
Focus order is not logical after expanding the 'Comp terms' on the 'Terms of use' page.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#20]
Attach (recommended) or Link to PDF file here:
https://mozilla.github.io/pdf.js/web/viewer.html.
[edad (2).pdf](https://github.com/gdh1995/pdf.js/files/14600520/edad.2.pdf)

Configuration:
- Operating system and its version: iPhone 13 Version-17.4
- PDF.js version: 1.4   

Pre-requisites:
Open Settings> Accessibility> Voiceover> Turn on Voiceover.

Steps to reproduce the problem:

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Navigate through the PDF controls (Find in doc, zoom in/out etc..) using keyboard.
3. Navigate to 'Toggle sidebar' and activate it.
4. Navigate through the appeared controls.
5. Observe the issue.

What is the expected behavior? (add screenshot)
Focus order should be logical after expanding the 'Comp terms' on the 'Terms of use' page.

After activating the 'Comp Terms' Expand/Collapse button and navigating the toolbar controls, focus order should be in logical sequence using the right swipe gesture on the page as before 'tools' button focus first land on the 'zoom in/ zoom out' controls.

What went wrong? (add screenshot)
Focus order is not logical after expanding the 'Comp terms' on the 'Terms of use' page.

When the 'Comp Terms' Expand/Collapse button is activated and the toolbar controls are navigated, the focus order does not move in a logical sequence using the right swipe gesture on the page.

https://github.com/gdh1995/pdf.js/assets/93735775/f7480fed-fafd-437e-9a92-6064b5d9b30e

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.
